### PR TITLE
[9.x] Pusher authorized connections

### PR DESCRIPTION
**Pull request by rennokki**
Before merging:

- [x] https://github.com/laravel/framework/pull/42531
- [x] https://github.com/laravel/framework/pull/42664
- [x] https://github.com/laravel/echo/pull/340

### CODE REVIEW

File: broadcasting.md [hunk 1]
```diff
@@ -515,13 +516,13 @@ Private channels require you to authorize that the currently authenticated user
 <a name="defining-authorization-routes"></a>
 ### Defining Authorization Routes
 
-Thankfully, Laravel makes it easy to define the routes to respond to channel authorization requests. In the `App\Providers\BroadcastServiceProvider` included with your Laravel application, you will see a call to the `Broadcast::routes` method. This method will register the `/broadcasting/auth` route to handle authorization requests:
+Thankfully, Laravel makes it easy to define the routes to respond to channel authorization requests. In the `App\Providers\BroadcastServiceProvider` included with your Laravel application, you will see a call to the `Broadcast::channelAuthorizationRoutes` method. This method will register the `/broadcasting/auth` route to handle authorization requests:
```
Comment (Wulfheart) [inline broadcasting.md:519]: I would leave out the "thankfully" at the beginning. This is more the style of a blog post not the one of a technical document.

Comment (rennokki) [reply broadcasting.md:519]: It was copied from above.

**Comment by rennokki** [comment]
@taylorotwell

**Comment by rennokki** [comment]
@taylorotwell Missed that by accident.

**Comment by taylorotwell** [comment]
We probably need to be more specific about where exactly to call `Echo.signin()`

**Comment by driesvints** [comment]
@rennokki did you have time for this yet?

**Comment by rennokki** [comment]
@taylorotwell Fixed that.

**Comment by taylorotwell** [comment]
Honestly I'm going to table this documentation for now. It's not a super in-demand feature of Echo. I may revisit this to give it proper documentation in the future.

**Pull request closed without merging**